### ISSUE TITLE
[4.0] atum Logo not responsive to sidebar toggle button

### ISSUE THREAD

**Issue by himanshu007-creator** [joomla/joomla-cms#33660]
### Steps to reproduce the issue
Login to backend and toggle the sidebar menu view. Notice the logo above.

### Expected result
It should change to Large/small logo instantly as the menu view changes.

### Actual result
It does not change instantly. Only after refreshing current page, changes are visible.
![jadu](https://user-images.githubusercontent.com/65963997/117550259-00650680-b05d-11eb-88bc-b60f7e435e36.gif)

### System information (as much as possible)
Joomla! 4.0.0-beta8-dev

**Comment by ghost** [comment]
Confirming issue.

**Comment by PhilETaylor** [comment]
It used to work like this... confirming that its now broken. 
https://user-images.githubusercontent.com/400092/117570071-b9a8f800-b0c0-11eb-8c39-145d93acc9b3.mp4

**Comment by PhilETaylor** [comment]
i think this is now fixed by another PR that was merged -please test again

**Comment by richard67** [comment]
Closing as having a pull request which is already RTC (ready to commit). See #33754 . Feel free to re-open if that PR doesn't solve all aspects of your issue.
